ios-base/Networking/TargetType+Base.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TargetType+Base.swift
+//  ios-base
+//
+//  Created by Mauricio Cousillas on 6/4/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import Moya
+
+// MARK: TargetType base configuration
+extension TargetType {
+
+  var baseURL: URL {
+    let baseUrlString = Bundle.main.object(forInfoDictionaryKey: "Base URL") as? String ?? ""
+    return URL(string: baseUrlString)!
+  }
+
+  var headers: [String: String]? {
+    return getHeaders()
+  }
+
+  var sampleData: Data { return Data() }
+
+}
+
+// MARK: TargetType helpers
+extension TargetType {
+
+  static private var baseHeaders: [String: String] {
+    return [
+      HTTPHeader.accept.rawValue: "application/json",
+      HTTPHeader.contentType.rawValue: "application/json"
+    ]
+  }
+
+  public func getHeaders() -> [String: String]? {
```

Comment:
would one be able to set custom headers for a `TargetType`?

Reply:
Yes! you just override the `headers` variable and return what you need, this function is just a helper to set some defaults.